### PR TITLE
(ref) Further Abstraction of PR #9

### PR DESCRIPTION
**Pull request by YvanCywan**
Further abstraction of the code in PR #9
Also includes an update to .github involving a new assign function for issues.

The controls functions I extracted could be moved into the core and setup as a function since they could easily be used elsewhere.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [no] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

File: client/interactions.lua [hunk 1]
```diff
@@ -42,65 +42,75 @@ local function GetCuffedAnimation(playerId)
...
-
-        if (not IsEntityPlayingAnim(cache.ped, "mp_arresting", "idle", 3) and not IsEntityPlayingAnim(cache.ped, "mp_arrest_paired", "crook_p2_back_right", 3)) and not PlayerData.metadata.isdead then
-            lib.requestAnimDict("mp_arresting")
-            TaskPlayAnim(cache.ped, "mp_arresting", "idle", 8.0, -8, -1, cuffType, 0, false, false, false)
-        end
+    if not PlayerData.metadata.ishandcuffed then return sleep end
+    sleep = 0
+    HandcuffActions()
+    if PlayerData.metadata.isdead then return sleep end
+    for _, anim in anim do
```
[the remark this answers is not in the document]
Comment (YvanCywan) [reply client/interactions.lua]: Done

**Comment by BerkieBb** [comment]
Actually, doing
```lua
for i = 1, #anim do
    local data = anim[i]
    -- rest of code
end
```
Would be better

**Comment by BerkieBb** [comment]
And would also fix the lint issue

**Comment by YvanCywan** [comment]
> Actually, doing
> 
> ```lua
> for i = 1, #anim do
>     local data = anim[i]
>     -- rest of code
> end
> ```
> 
> Would be better

Should be better now.

**Pull request merged**